behat/features/threshold.feature

```diff
@@ -0,0 +1,40 @@
+Feature: Check threshold functionally
+  In order to be able to create CI build items with error state
+  As a privileges user
+  I need to see new status of CI build and new incidents.
+
+  @api
+  Scenario: Check create new repository.
+    Given I login with user "William"
+    When  I create repository "William/app"
+    And   I visit repository "William/app"
+    Then  I should have access to the page
+
+  @api
```

Comment:
+ @api
+  Scenario: Check create new CI build for new created repository.
+    Given I login with user "William"
+    When I create CI build "William/app1" for repository "William/app1"
+    And   I create "2" CI build items with status "Done" for CI build "William/app"
 Then CI build "William/app1" incident status should be "ok"
```